src/main/java/gregtech/integration/jei/multiblock/infos/SteamOvenInfo.java

```diff
@@ -0,0 +1,66 @@
+package gregtech.integration.jei.multiblock.infos;
+
+import com.google.common.collect.Lists;
+import gregtech.api.metatileentity.multiblock.MultiblockControllerBase;
+import gregtech.common.ConfigHolder;
+import gregtech.common.blocks.BlockFireboxCasing;
+import gregtech.common.blocks.BlockMetalCasing;
+import gregtech.common.blocks.MetaBlocks;
+import gregtech.common.metatileentities.MetaTileEntities;
+import gregtech.integration.jei.multiblock.MultiblockInfoPage;
+import gregtech.integration.jei.multiblock.MultiblockShapeInfo;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.EnumFacing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SteamOvenInfo extends MultiblockInfoPage {
+
+    @Override
+    public MultiblockControllerBase getController() {
+        return MetaTileEntities.STEAM_OVEN;
+    }
+
+    @Override
+    public List<MultiblockShapeInfo> getMatchingShapes() {
+        ArrayList<MultiblockShapeInfo> shapeInfo = new ArrayList<>();
+
+        if (ConfigHolder.steelMultiblocks) {
+            shapeInfo.add(MultiblockShapeInfo.builder()
+                    .aisle("FFF", "IXX", "###")
+                    .aisle("HFF", "S#X", "XXX")
+                    .aisle("FFF", "OXX", "###")
+                    .where('S', MetaTileEntities.STEAM_OVEN, EnumFacing.WEST)
+                    .where('X',  MetaBlocks.METAL_CASING.getState(BlockMetalCasing.MetalCasingType.STEEL_SOLID))
+                    .where('F', MetaBlocks.BOILER_FIREBOX_CASING.getState(BlockFireboxCasing.FireboxCasingType.STEEL_FIREBOX))
+                    .where('I', MetaTileEntities.STEAM_IMPORT_BUS, EnumFacing.WEST)
+                    .where('O', MetaTileEntities.STEAM_EXPORT_BUS, EnumFacing.WEST)
```

Comment:
I think these should allow any hatch, or at least ULV and LV

Reply:
hmm, maybe

Reply:
From Discord:

> It was intended to be only steam

Reply:
Ok